libvmi/driver/kvm/libvirt_wrapper.c

```diff
@@ -55,6 +56,18 @@ status_t create_libvirt_wrapper(kvm_instance_t *kvm)
         }
     }
 
+    wrapper->handle_qemu = dlopen ("libvirt-qemu.so", RTLD_NOW | RTLD_GLOBAL);
+
+    if ( !wrapper->handle_qemu ) {
+        // fallback to libvirt-qemu.so.0
+        wrapper->handle_qemu = dlopen ("libvirt-qemu.so.0", RTLD_NOW | RTLD_GLOBAL);
+
+        if ( !wrapper->handle_qemu ) {
+            dbprint(VMI_DEBUG_KVM, "--failed to open a handle to libvirt-qemu\n");
```

Comment:
We need to `dlclose` the `libvirt` handle here in case of failure

Reply:
took remark into account